Mathlib/Analysis/SpecialFunctions/PolarCoord.lean

```diff
@@ -89,20 +89,24 @@ def polarCoord : PartialHomeomorph (ℝ × ℝ) (ℝ × ℝ) where
     · exact (Complex.continuousAt_arg hz).continuousWithinAt
     · exact Complex.equivRealProdCLM.symm.continuous.continuousOn
 
+/-- The derivative of `polarCoord.symm`, see `hasFDerivAt_polarCoord_symm`. -/
+def FDerivPolarCoordSymm : ℝ × ℝ → ℝ × ℝ →L[ℝ] ℝ × ℝ :=
```

Comment:
```suggestion
def fderivPolarCoordSymm (p : ℝ × ℝ) : ℝ × ℝ →L[ℝ] ℝ × ℝ :=
```
Since it doesn't return a type, it should be lower capitalized. Also, move the argument left of the colon for readability.

Reply:
Done. I was not sure about the naming of this one. Thanks for the explanation.